requirements/base.in

```diff
@@ -24,10 +24,11 @@ mozilla-django-oidc-db
 
 # API libraries
 djangorestframework
-gemma-zds-client<1.0
+gemma-zds-client<1.0P
```

Comment:
Why is this change?

Reply:
Oh, i didn't see it, it should be a miss type, Shift + P instead of Ctrl + P ^^'
Thanks !